src/LearnMe.Core/Services/Calendar/Utils/Constants/ApplicationConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LearnMe.Core.Services.Calendar.Utils.Constants
+{
+    public static class ApplicationConstants
+    {
+        public const string ApplicationName = "Learn Me WEB Applicaton";
+
+        public const int LastSynchronizationRecordId = 1;
```

Comment:
consta nie zinkrementujesz

Reply:
Zresztą, tutaj bym widział jakąś tabelę z synchronizacjami. Np tabela z kolumnami takimi jak data synchronizacji, kto ją przeprowadził, moze jakiś komentarz albo ile danych zsynchronizowano?

Reply:
Absolutnie się zgadzam :) tutaj mam tylko zapis, kiedy była ostatnia synchronizacja - tylko że to jest w bazie dlatego w formie tabelki z dosłownie 1 wierszem (Id = 1 zawsze) i updatuję datę i godzinę jeśli synchronizacja przebiegła pomyślnie. Myślę że na razie tak zostawię.